spec/controllers/state_file/questions/data_review_controller_spec.rb

```diff
@@ -18,7 +18,7 @@
         expect(efile_info.ip_address.to_s).to eq "72.34.67.178"
         expect(efile_info.device_id).to eq nil
         expect(efile_info.intake).to eq intake
-        expect(response).to redirect_to("/en/az/questions/name-dob")
+        expect(response).to be_ok
```

Comment:
I would prefer the more specific condition `render_template :edit` instead of `be_ok`

Reply:
Done